src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.tsx

```diff
@@ -105,8 +106,8 @@ export const RoomCard: React.FC<RoomCardProps> = ({
           {recentRoomUsers.length}
         </div>
         <div className="RoomCard__type">{room.template}</div>
-        <div className="RoomCard__link">
-          <PrettyLink to={room.url} />
+        <div className="RoomCard__link" onClick={() => openRoomUrl(room.url)}>
+          {room.url}
```

Comment:
`PrettyLink` has styles tuned to the design spec from https://www.figma.com/file/1Lv4r46oF1ioHZ6a9JRaRR/Admin?node-id=0%3A1
![sparkle-pretty-link-01](https://user-images.githubusercontent.com/79229621/128658984-be7d7f93-ab12-434a-a43a-6d50d1d8739c.png)


Without `PrettyLink`:
![sparkle-pretty-link-02](https://user-images.githubusercontent.com/79229621/128659743-d38d1e54-9cb9-424b-a537-e35bab66ed18.png)


With `PrettyLink`:
![sparkle-pretty-link-03](https://user-images.githubusercontent.com/79229621/128659754-9fe79634-77d1-4c58-a717-5574d55384a4.PNG)


A way to have the "best of both worlds":
```typescript
// In RoomCard.tsx
<div className="RoomCard__link">
  <PrettyLink to={room.url} />
</div>
```

```typescript
// In PrettyLink.tsx
<div className="PrettyLink PrettyLink__wrapper"  onClick={() => openRoomUrl(props.to)}>
    <span className="PrettyLink PrettyLink__text" title={title}>
      {text}
    </span>
</div>
```

Reply:
ping @denisdimitrov

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/a3835da5ae0f7d54afb9effca82fd8391cc61837

Reply:
I had to pass `onClick` to the component to make it flexible and avoid encapsulating `openRoomUrl` inside the component.

Reply:
Better that way. This should ideally be a button, an `ButtonNG` but kind of OOS ATM